packages/calcite-components/conventions/Documentation.md

```diff
@@ -105,6 +105,23 @@ export default {
 - The `simple` story and any other significant configuration should have a matching `darkModeRTL_TestOnly` story
   - In order to reduce snapshot count, dark mode and RTL visual tests have been combined
 
+Any stories used for snapshot testing should have animations disabled using the `--calcite-duration-factor` CSS variable so they don't cause diffs on every build. `NoTest` stories are the only ones that should have animations. See [`loader`'s stories](../src/components/loader/loader.stories.ts) as an example for how to setup stories for components with animation. For example:
```

Comment:
What about the stories that don't have a test-related suffix? I believe those are also used for screenshot tests.

I think we can automatically disable all animations in Chromatic builds by using [preview/manager templates](https://storybook.js.org/docs/6.5/html/addons/writing-presets#previewmanager-templates) + the screenshot test build flag:

```ts
// main.ts
module.exports = {
  // ...
  previewHead: (head) => (`
    ${head}
    ${
      process.env.STORYBOOK_SCREENSHOT_TEST_BUILD ? 
        `<style>
          :root {
            --calcite-duration-factor: 0;
          }
        </style>` 
        : ""  
    }
  `),
};
```

Reply:
> What about the stories that don't have a test-related suffix? I believe those are also used for screenshot tests.

Yeah they are.

Ooh nice I can give that a try

Reply:
There is also this: https://www.chromatic.com/docs/ischromatic#use-in-storybookpreviewjs


 I just didn't know about the previeHead thing